projects/packages/sync/src/modules/class-posts.php

```diff
@@ -597,6 +597,9 @@ public function filter_post_content_and_add_links( $post_object ) {
 	 * @param \WP_Post $post       Post object.
 	 */
 	public function save_published( $new_status, $old_status, $post ) {
+		if ( ! $post instanceof \WP_Post || ! isset( $post->ID ) ) {
```

Comment:
Isn't the check for `$post->ID` redundant here, as a `WP_Post` object would always have an ID?
```suggestion
		if ( ! $post instanceof \WP_Post ) {
```